serenity/ema_map.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef SERENITY_EMA_MAP_HPP
+#define SERENITY_EMA_MAP_HPP
+
+#include <string>
+
+#include <unordered_map>
+
+#include "filters/ema.hpp"
+
+#include "mesos/mesos.hpp"
+
+
+namespace mesos {
+namespace serenity {
+
+/**
+ * Hasher functor for EmaMap
+ */
+struct EmaMapHasher{
+  size_t operator()(const ExecutorInfo& that) const {
+    std::string hashKey = that.executor_id().value() +
+                          that.framework_id().value();
+    std::hash<std::string> hashFunc;
+    return hashFunc(hashKey);
+  }
+};
+
+
+/**
+ * Equals functor for EmaMap
+ */
+struct EmaMapEquals {
```

Comment:
You should name it ExecutorInfoEquals and this piece of code should be reusable in other components.